psdk_wrapper/src/modules/telemetry.cpp

```diff
@@ -834,6 +841,36 @@ PSDKWrapper::rc_callback(const uint8_t *data, uint16_t data_size,
   return DJI_ERROR_SYSTEM_MODULE_CODE_SUCCESS;
 }
 
+T_DjiReturnCode
+PSDKWrapper::esc_callback(const uint8_t *data, uint16_t data_size,
+                          const T_DjiDataTimestamp *timestamp)
+{
+  (void)data_size;
+  (void)timestamp;
+  std::unique_ptr<T_DjiFcSubscriptionEscData> esc_data =
+      std::make_unique<T_DjiFcSubscriptionEscData>(
+          *reinterpret_cast<const T_DjiFcSubscriptionEscData *>(data));
+  psdk_interfaces::msg::EscData esc_msg;
+  esc_msg.header.stamp = this->get_clock()->now();
+  // Populate the message with ESC data
+  for (int i = 0; i < 4; ++i)
```

Comment:
I think to make it as generic as possible, we should fill all the data available in here. Even though most cases will be quadcopters, as this wrappers intends to be as generic as possible, we should fill all the vector.